cypress-tests/cypress/e2e/PaymentUtils/CardPayments.js

```diff
@@ -0,0 +1,995 @@
+        body: {
+          // Assuming Bluesnap returns "requires_customer_action" for 3DS authentication
+          status: "requires_customer_action",
+          // You'll likely get some specific Bluesnap data here, replace with appropriate values
+          bluesnap_data: {
+            // ... Bluesnap specific data
+          },
+        },
+      },
+    },
+    "Bluesnap3DSAutoCapture": {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: {
+            card_number: "4000000000002503",
+            card_exp_month: "01",
+            card_exp_year: "25",
+            card_holder_name: "joseph Doe",
+            card_cvc: "123",
+          },
+        },
+        currency: "USD",
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
+      },
+      Response: {
+        status: 200,
+        body: {
+          // Assuming Bluesnap returns "requires_customer_action" for 3DS authentication
+          status: "requires_customer_action",
+          // You'll likely get some specific Bluesnap data here, replace with appropriate values
+          bluesnap_data: {
+            // ... Bluesnap specific data
+          },
+        },
+      },
+    },
+    "BluesnapNo3DSManualCapture": {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: {
+            card_number: "4242424242424242",
+            card_exp_month: "01",
+            card_exp_year: "25",
+            card_holder_name: "joseph Doe",
+            card_cvc: "123",
+          },
+        },
+        currency: "USD",
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
+      },
+      Response: {
+        status: 200,
+        body: {
+          // Assuming Bluesnap returns "requires_capture" for non-3DS transactions
+          status: "requires_capture",
+          // You'll likely get some specific Bluesnap data here, replace with appropriate values
+          bluesnap_data: {
+            // ... Bluesnap specific data
+          },
+        },
+      },
+    },
+    "BluesnapNo3DSAutoCapture": {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: {
+            card_number: "4242424242424242",
+            card_exp_month: "01",
+            card_exp_year: "25",
+            card_holder_name: "joseph Doe",
+            card_cvc: "123",
+          },
+        },
+        currency: "USD",
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
+      },
+      Response: {
+        status: 200,
+        body: {
+          // Assuming Bluesnap returns "succeeded" for non-3DS transactions
+          status: "succeeded",
+          // You'll likely get some specific Bluesnap data here, replace with appropriate values
+          bluesnap_data: {
+            // ... Bluesnap specific data
+          },
+        },
+      },
+    },
+  },
+  bank_transfer_pm: {
```

Comment:
assigned issue asks not to add configs for payment methods other than cards. so, this entire section can be removed:

`bank_transfer_pm`
`bank_redirect_pm`
`upi_pm`